src/python-common/ceph/deployment/service_spec.py

```diff
@@ -1765,12 +1765,13 @@ def __init__(self,
                  config: Optional[Dict[str, str]] = None,
                  networks: Optional[List[str]] = None,
                  placement: Optional[PlacementSpec] = None,
-                 frontend_port: Optional[int] = 80,
-                 ssl_cert: Optional[str] = None,
-                 ssl_key: Optional[str] = None,
-                 ssl_dh_param: Optional[str] = None,
+                 disable_https: Optional[bool] = False,
+                 port: Optional[int] = None,
+                 ssl_certificate: Optional[str] = None,
+                 ssl_certificate_key: Optional[str] = None,
+                 ssl_prefer_server_ciphers: Optional[str] = "on",
```

Comment:
mmm really good question! since I'm literally mimicking `nginx` configuration I though it's a good idea to stick to their same option so make it more user friendly. Otherwise we will end up with a mix of bool options + nginx way to handle boolean options (no/off) ... WDYT?

Reply:
It's only more user friendly to use the direct nginx options if we expect users to be familiar with nginx configuration. I don't think it's a huge deal either way so I'm fine with leaving it as it is, but I do think there's an argument for trying to hide aspects of the nginx configuration behind settings that might be easier to not  mess up (I think it's easier to set the bool properly than to know it should be the string "on" or "off")